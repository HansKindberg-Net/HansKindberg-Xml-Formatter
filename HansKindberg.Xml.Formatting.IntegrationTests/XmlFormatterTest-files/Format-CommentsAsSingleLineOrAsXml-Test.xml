<?xml version="1.0" encoding="utf-8"?>
<root><!--<add type="first-type,first-assembly" name="first"></add>--><!--<collection><add type="second-type,second-assembly" name="second"></add><add type="first-type,first-assembly" name="first"></add></collection>--><!--Comment<collection><add type="second-type,second-assembly" name="second"></add><add type="first-type,first-assembly" name="first"></add></collection>--></root>