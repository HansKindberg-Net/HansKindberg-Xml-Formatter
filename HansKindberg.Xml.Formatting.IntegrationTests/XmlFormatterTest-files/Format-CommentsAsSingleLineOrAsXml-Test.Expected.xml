<?xml version="1.0" encoding="utf-8"?>
<root>
	<!-- <add name="first" type="first-type, first-assembly" /> -->
	<!--
		<collection>
			<add name="first" type="first-type, first-assembly" />
			<add name="second" type="second-type, second-assembly" />
		</collection>
	-->
	<!--
		Comment
		<collection>
			<add name="first" type="first-type, first-assembly" />
			<add name="second" type="second-type, second-assembly" />
		</collection>
	-->
</root>